lib/get-user-numbers-2012.rb

```diff
@@ -1,24 +1,18 @@
 require 'zendesk_api'
 
-@client = ZendeskAPI::Client.new do |config|
-  config.url = ENV['ZENDESK_URL']
-  config.username = ENV['ZENDESK_USER_EMAIL']
-  config.password = ENV['ZENDESK_USER_PASSWORD']
-
-  config.retry = true
-end
+require_relative 'zendesk-setup.rb'
 
 #
 # 2012
 #
 
-@y2012_users = []
+@y2012_tickets = []
 
 (1..140).each do |i|
-  @client.search(:query => "type:user created_at>2012-01-01 created_at<2013-01-01 role:end-user -name:Zendesk organization_id:none").page(i).each do |user|
-    @y2012_users << user['id']
-    puts user['id']
+  @client.search(:query => "type:ticket group_id:#{group_id} organization_id:none status:closed updated_at>=2012-01-01 updated_at<2013-01-01").page(i).each do |ticket|
```

Comment:
This code gets ticket ids instead of user ids. It doesn't seem to match the name of script `get-user-numbers-2012.rb `